<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>461</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Профиль</string>
    </attribute>
    <widget class="QLabel" name="admin_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>   Пациенты  </string>
    </attribute>
    <widget class="QTableWidget" name="patient_table">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>331</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Логин</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пароль</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pat_del">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>202</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>202</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>   Врачи  </string>
    </attribute>
    <widget class="QTableWidget" name="docs_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>49</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>39</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>32</number>
     </attribute>
     <column>
      <property name="text">
       <string>Логин</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Должность и описание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="doc_del">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_doctor">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить врача</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restart">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>  Диагнозы </string>
    </attribute>
    <widget class="QTableWidget" name="diagz_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>431</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Логин</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Логин врача</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Имя врача</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Проблема</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Диагноз</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="diag_del">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Персоны</string>
    </attribute>
    <widget class="QTableWidget" name="who_table">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>256</width>
       <height>211</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Кто</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
